ambari-server/src/main/resources/stack-hooks/before-START/scripts/params.py

```diff
@@ -269,7 +269,7 @@
   
 ambari_libs_dir = "/var/lib/ambari-agent/lib"
 is_webhdfs_enabled = config['configurations']['hdfs-site']['dfs.webhdfs.enabled']
-default_fs = config['configurations']['core-site']['fs.defaultFS']
+default_fs = default('/configurations/core-site/fs.defaultFS',None)
```

Comment:
Can you please explain the need for this change?  Please include steps to reproduce the problem it is trying to solve.